private/underlay/conn/conn_spec.gobra

```diff
@@ -65,46 +62,11 @@ pred (c *Config) Mem() {
 /** Lift methods in *connUDPBase to *connUDPIPv4 **/
 *connUDPIPv4 implements Conn
 
-requires  acc(c.Mem(), _)
-preserves sl.Bytes(b, 0, len(b))
-ensures   err == nil ==> 0 <= n && n <= len(b)
-ensures   err == nil ==> acc(addr.Mem(), _)
-ensures   err != nil ==> err.ErrorMem()
-func (c *connUDPIPv4) ReadFrom(b []byte) (n int, addr *net.UDPAddr, err error) {
-	unfold acc(c.Mem(), _)
-	unfold acc(c.connUDPBase.MemWithoutConn(), _)
-	assert c.pconn.GetUnderlyingConn() == c.conn
-	tmpImpl := c.conn
-	tmpItf := c.pconn.ExchangeWildcardPerm()
-	var packetconn *ipv4.PacketConn = c.pconn
-	fold acc(c.connUDPBase.Mem(), _)
-	n1, addr1, err1 := c.connUDPBase.ReadFrom(b, tmpImpl)
-	return n1, addr1, err1
-}
-
-preserves acc(c.Mem(), _)
-preserves acc(sl.Bytes(b, 0, len(b)), R15)
-ensures   err == nil ==> 0 <= n && n <= len(b)
-ensures   err != nil ==> err.ErrorMem()
-func (c *connUDPIPv4) Write(b []byte) (n int, err error) {
-	unfold acc(c.Mem(), _)
-	unfold acc(c.connUDPBase.MemWithoutConn(), _)
-	assert c.pconn.GetUnderlyingConn() == c.conn
-	tmpImpl := c.conn
-	tmpItf := c.pconn.ExchangeWildcardPerm()
-	var packetconn *ipv4.PacketConn = c.pconn
-	assert tmpItf == c.conn
-	fold acc(c.connUDPBase.Mem(), _)
-	n1, err1 := c.connUDPBase.Write(b, tmpImpl)
-	return n1, err1
-}
```

Comment:
why did we drop these functions instead of rewriting them?

Reply:
Both `ReadFrom` and `Write` are no longer part of the interface `Conn`. The functions are also not used in the package itself.

I therefore assumed that the spec is no longer needed. Is this not the case?

Reply:
I just noticed, that the `Write` function still exists, just not as part of the `Conn` interface. I will readd it.

Reply:
Done.